src/client/content_cao.cpp

```diff
@@ -1528,19 +1529,52 @@ void GenericCAO::updateAnimationSpeed()
 	m_animated_meshnode->setAnimationSpeed(m_animation_speed);
 }
 
-void GenericCAO::updateBonePosition()
+void GenericCAO::updateBonePosition(f32 dtime)
 {
-	if (m_bone_position.empty() || !m_animated_meshnode)
+	if (!m_animated_meshnode)
+		return;
+	if (m_bone_override.empty()) {
+		m_animated_meshnode->setJointMode(scene::EJUOR_NONE);
 		return;
+	}
 
 	m_animated_meshnode->setJointMode(scene::EJUOR_CONTROL); // To write positions to the mesh on render
-	for (auto &it : m_bone_position) {
+	for (auto &it : m_bone_override) {
 		std::string bone_name = it.first;
 		scene::IBoneSceneNode* bone = m_animated_meshnode->getJointNode(bone_name.c_str());
-		if (bone) {
-			bone->setPosition(it.second.X);
-			bone->setRotation(it.second.Y);
+		if (!bone)
+			continue;
+
+		BoneOverride *override = it.second;
```

Comment:
```suggestion
		const BoneOverride *spec = it.second;
```
1. To avoid the `override` keyword for syntax highlighting reasons
2. Const to keep the code reasonable and more self-explanatory.